### DISCUSSION TITLE
Larger description textbox

### DISCUSSION THREAD

**Discussion by robosong** [getredash/redash#6432] [Ideas]
In query edit mode, please keep the description text box expanded to fit the contents when I click to edit the description. Currently when I click to edit the description the text box shrinks to display only two lines which clips the contents and makes it very annoying to edit. I always have to manually expand it to see the full contents again.

Thanks.

**Comment by guidopetri** [comment]
Thanks for pointing out this bug @robosong , I created an issue from it in #6434 . Closing this discussion so we don't have it marked in two spots
